ports/onnxruntime/project-config-template.cmake

```
@@ -0,0 +1,50 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+# sync with portfile.cmake and onnxruntime_vcpkg_deps.cmake ...
+find_dependency(absl REQUIRED)
+find_dependency(re2 REQUIRED)
+if(@onnxruntime_BUILD_UNIT_TESTS@)
```

Generally I would recommend quotes 
```suggestion
if("@onnxruntime_BUILD_UNIT_TESTS@")
```
so that ON/OFF are never dereferenced as variables, and empty substitutions still leave valid cmake syntax.

---

Nice. I will apply them soon